gulpfile.js/tasks/build.js

```diff
@@ -28,7 +29,7 @@ const buildSass = () => {
     )
     .pipe(rename({ basename: 'standalone' }))
     .pipe(dest(outDir))
-    .pipe(postcss([cssnano()]))
+    .pipe(postcss([cssnano(), postcssPresetEnv()]))
```

Comment:
@Josh68 Can we make it read from our postcss.config.js so we have one shared postcss config for both storybook/webpack and gulp?

Reply:
Yeah, I was thinking that. It's not ready (hence, still in draft). I need to condition `cssnano`, I think.

Reply:
If I add an extra property name/value pair in the mixin, I get output 🤔 

```css
 &-inline-start-#{$name} {
    position: relative;
    @if $property == margin {
      margin-inline-start: $value;
    } @else {
      padding-inline-start: $value;
    }
  }
```

```css
.u-pad-inline-start-1 {
    position: relative;
}
```

Reply:
This is still a WIP with issues remaining in some examples. Using the `preserve` option for the PostCSS logical properties module, some properties are being left as-is and some converted to directional properties. There may be a bug in the module, but I'm not entirely clear on that yet.

Ideally, everything would be converted with fallbacks for non-conforming browsers. There may also be a need to refactor with `dir` attributes set on parent elements for each example. However, the current issues are showing up with logical block properties (which appear to be converted only to directional properties, without preserving the logical versions), and I'm not clear whether `dir` makes a difference in these cases. Adding extra markup throws off existing demo CSS, so I've abandoned that approach for the moment.